src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -37,10 +37,8 @@
 
 FEATURE_EXTRACTOR_MAPPING_NAMES = OrderedDict(
     [
-        ("audio-spectrogram-transformer", "ASTFeatureExtractor"),
```

Comment:
For the entirety of the file, you should not have any deletion. I think that if you rebase you should be able to get rid of this.